core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -59,7 +87,17 @@ oppia.factory('explorationData', [
             $log.info('Retrieved exploration data.');
             $log.info(response.data);
 
+            // Initialize the changeList by the one received from the backend.
             explorationData.data = response.data;
+            changeListService.loadAutosavedChangeList(
```

Comment:
@526avijitgupta btw, I discussed this with @jacobdavis11, and I think he agrees with the fix